libs/designsystem/src/lib/components/segmented-control/segmented-control.component.spec.ts

```diff
@@ -40,8 +31,12 @@ describe('SegmentedControlComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      schemas: [NO_ERRORS_SCHEMA],
-      declarations: [SegmentedControlComponent, BadgeComponent, MockIonSegmentButtonComponent],
+      declarations: [
```

Comment:
I don't know if this is a general approach we want to take, but hear me out (sit down, grab a bowl of popcorn and read on):

Whenever touching upon an existing test-suite (case, whatever it's called) using the "raw" `TestBed`, we should refactor to use `@ngneat/spectator`. That way we'll:

- Align our (test) code
- Avoid boilerplate
- Have more fun (okay, I made that one up...)

Not sure if you want to do that now, or later on... or think it's a bad idea... it's just nice to align code (or so I think).

If you decide to do so, I would highly recommend adding the following `tslint`-rule:

```json
...
"ban": [
  true,
  ["TestBed", "*", "Use 'Spectator' instead"]
]
...
```

Reply:
I totally agree! And that's excactly the approach I've taken, so if you see the last 5-8 specs changes, they are changed from Testbed to Spectator 👍